tests/golang.rs

```diff
@@ -24,7 +25,92 @@ macro_rules! tests {
 
         // https://github.com/golang/go/blob/master/test/chan/doubleselect.go
         mod doubleselect {
-            // TODO
+            use super::*;
+
+            const ITERATIONS: usize = 100_000;
+
+            #[test]
+            #[ignore]
+            fn test_double_select() {
+                let (c1, r1) = channel::unbounded();
```

Comment:
To be perfectly pedantic, `make(chan int)` should be translated into `channel::unbounded::<i32>()` since `int` is equivalent to `i32`.

But I'm just nitpicking now... you don't have to do this, but can if you want to. :)

Reply:
This should be handled by making `ITERATIONS` an `i32` right?